server/master/src/main/java/org/apache/accumulo/master/metrics/fate/FateLegacyJMXMetricsMBean.java

```diff
@@ -16,12 +16,12 @@
  */
 package org.apache.accumulo.master.metrics.fate;
 
-public interface FateMetricsMBean {
+public interface FateLegacyJMXMetricsMBean {
```

Comment:
This was a rename so that on the merge forward, these JMX specific classes are drooped and hadoop2 metrics classes the can be named FateMetrics.  The interface name needed to change to match the JMX required naming convention